.github/workflows/ci.yml

```
@@ -174,7 +174,7 @@ jobs:
           save-if: ${{ github.event_name != 'merge_group' }}
 
       - name: cargo nextest
-        run: cargo nextest run --workspace --locked --all-features --no-fail-fast
+        run: cargo nextest run --workspace --locked --all-features --no-fail-fast --no-tests=warn
```

Interesting change with `nextest`. Now `warn` works fine, but I'm thinking we should just go with `pass` to mimic vanilla `cargo test`. The output already shows the number of tests - zero - and we consider that passable.

```suggestion
        run: cargo nextest run --workspace --locked --all-features --no-fail-fast --no-tests=pass
```

---

Done.